palm/plugins/dbt/commands/cmd_model.py

```diff
@@ -89,12 +77,12 @@ def create_model(model_name, model_type, ctx, use_ref_file):
     click.echo(result)
 
     if use_ref_file:
-        sql_to_dbt.create_dbt_sql_file(model_name, model_type)
+        sql_to_dbt.create_dbt_sql_file(model_name, models_path)
 
 
-def create_docs(model_name, model_type, ctx, use_ref_file):
+def create_docs(model_name, model_dir, model_type, ctx, use_ref_file):
     docs_template_path = Path(Path(__file__).parent.parent, 'model_template', 'docs')
-    docs_path = Path('models/arbor/documentation/models/', model_type)
+    docs_path = Path('models/', model_dir, '/documentation/models/', model_type)
```

Comment:
arguably we should parameterize the whole path and not insist that people follow our convention

Reply:
Same thing for docs as what I just said above. We should remove `'/documentation/models'` from the docs path. This still imposes our convention. If anything, we might want another flag for docs path, otherwise I would default to putting the docs in the same place as the model (`models/<model-dir>/docs/`), or reverse the path written here and do `models/documentation/<model-dir>` so docs are navigable from the top level.